src/lib/components/helpToolTip/helpToolTip.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../styles/variables/variables.module.scss';
+@import '../../styles/themes/themes.module.scss';
+@import './reactPopperTooltip.module.scss';
+
+.questionMarkIcon {
+  cursor: pointer;
+}
+
+.questionMarkIcon:hover {
```

Comment:
You can use `&:hover {` inside questionMarkIcon, instead of repeating the class, because it is scss...it is not an error anyway